db/schema.rb

```diff
@@ -0,0 +1,43 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `rails
+# db:schema:load`. When creating a new database, `rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2020_06_03_212021) do
+
+  create_table "attendances", force: :cascade do |t|
+    t.integer "user_id"
```

Comment:
#### Migrations

* [ ] events table has creator_id as the foreign key ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

- Even though this won't change the functionality of your project, this was required to practice the use of foreign keys, please add the migration and correct your model as well

Reply:
The creator of the events has already been renamed to "Creator" in the event model. At no point in the OdinProject description has it been specified that it should be renamed in the table itself.